expression/generator/time_vec.go

```diff
@@ -524,6 +524,68 @@ func (b *{{.SigName}}) vectorized() bool {
 
 var addOrSubDate = template.Must(template.New("").Parse(`
 {{ range .Sigs }}
+{{- if eq .TypeA.TypeName "String"}}
+func (b *{{.SigName}}) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
+n := input.NumRows()
+	unit, isNull, err := b.args[2].EvalString(b.ctx, chunk.Row{})
+	if err != nil {
+		return err
+	}
+	if isNull {
+		result.ResizeTime(n, true)
```

Comment:
I think `vecEvalString` should return string column, why return time column here?

Reply:
done